lit_nlp/examples/models/tydi.py

```diff
@@ -0,0 +1,176 @@
+  FIELD_RENAMES = {
+      "input_text": "context",
+      "target_text": "question",
+  }
+  @property
+  def max_seq_length(self):
+    return self.model.config.max_position_embeddings
+
+  def __init__(self, 
+              model_name="mrm8488/bert-multi-cased-finedtuned-xquad-tydiqa-goldp", 
+              model=FlaxBertForQuestionAnswering.from_pretrained("mrm8488/bert-multi-cased-finedtuned-xquad-tydiqa-goldp"),
+              tokenizer=BertTokenizer.from_pretrained("mrm8488/bert-multi-cased-finedtuned-xquad-tydiqa-goldp"),
+              **config_kw):
+    super().__init__()
+    self.config = T5ModelConfig(**config_kw)
+    self.tokenizer = tokenizer or transformers.AutoTokenizer.from_pretrained(
+        model_name, use_fast=False)
+    # TODO(lit-dev): switch to TFBertForPreTraining to get the next-sentence
+    # prediction head as well.
+    self.model = model or model_utils.load_pretrained(
+        transformers.TFBertForMaskedLM,
+        model_name,
+        output_hidden_states=True,
+        output_attentions=True)
+
+    # # TODO(gehrmann): temp solution for ROUGE.
+    self._scorer = rouge_scorer.RougeScorer(["rougeL"], use_stemmer=True)
+    # If output is List[(str, score)] instead of just str
+    self._multi_output = isinstance(self.output_spec()["output_text"],
+                                    lit_types.GeneratedTextCandidates)
+    self._get_pred_string = (
+        lit_types.GeneratedTextCandidates.top_text if self._multi_output else
+        (lambda x: x))
+  ##
+  # LIT API implementation
+  def max_minibatch_size(self) -> int:
+    # The lit.Model base class handles batching automatically in the
+    # implementation of predict(), and uses this value as the batch size.
+    return self.config.inference_batch_size
+
+  def predict_minibatch(self, inputs):
+    """Predict on a single minibatch of examples."""
+    # tokenize the text. -> then return prediction
+  
+    # Text as sequence of sentencepiece ID"s.
+    context =[]
+    question = []
+    for i in inputs:
+        question.append(i['question'])
+        context.append(i['context'])
+
+    prediction_output = []
+    for i in range(len(inputs)):
+        model = FlaxBertForQuestionAnswering.from_pretrained("mrm8488/bert-multi-cased-finedtuned-xquad-tydiqa-goldp")
+        inputs = self.tokenizer(question[i], context[i], return_tensors="jax",padding=True)
+        outputs = model(**inputs)
+
+        answer_start_index = outputs.start_logits.argmax()
+
+        answer_end_index = outputs.end_logits.argmax()
+
+        predict_answer_tokens = inputs.input_ids[0, answer_start_index : answer_end_index + 1]
+
+        # creating output DIct
+        output = {
+            "output_text" : self.tokenizer.decode(predict_answer_tokens)
+        }
+        prediction_output.append(output)
+    
+    print('Predictions for the data is---->/n')
+    print(prediction_output)
+
+    # returning list of prediction
+    return prediction_output
```

Comment:
Looking over the example provided with the [`FlaxBertForQuestionAnswering.__call__(...)` docs](https://huggingface.co/transformers/v4.6.0/model_doc/bert.html#transformers.FlaxBertForQuestionAnswering.__call__), I think we need to change this implementation a bit... 
```suggestion
  def predict_minibatch(self, inputs):
    """Predict on a single minibatch of examples."""
    # tokenize the text. -> then return prediction
  
    # Text as sequence of sentencepiece ID"s.
    prediction_output = []
    for inp in inputs:
      tokenized = self.tokenizer(inp['question'], inp['context'], return_tensors="jax", padding=True)
      output = self.model(**tokenized)
      answer_start_index = output.start_logits.argmax()
      answer_end_index = output.end_logits.argmax()
      predict_answer_tokens = inputs.input_ids[0, answer_start_index : answer_end_index + 1]
      prediction_output.append({
          "output_text" : self.tokenizer.decode(predict_answer_tokens)
      })
    
    print('Predictions for the data is---->/n')
    print(prediction_output)

    # returning list of prediction
    return prediction_output
```